app.js

```diff
@@ -178,16 +193,22 @@ async function validateLogOnResume(opts) {
  *
  * @param {string} url - URL of database.
  * @param {object} opts - Options.
+ * @param {boolean} backup - true for backup, false for restore
  * @returns Boolean true if all checks are passing.
  */
-async function validateArgs(url, opts) {
+async function validateArgs(url, opts, backup = true) {
```

Comment:
Maybe `isBackup`? We use var name `backup` a lot, so it became is a bit vague

Reply:
good idea - I've changed it